qt/widgets/common/src/MuonFitDataSelector.cpp

```diff
@@ -356,7 +356,23 @@ QString MuonFitDataSelector::getSimultaneousFitLabel() const {
  * @param label :: [input] Text to set as label
  */
 void MuonFitDataSelector::setSimultaneousFitLabel(const QString &label) {
-  m_ui.txtSimFitLabel->setText(label);
+  // do some checks that it is valid
+  auto safeLabel = label.toStdString();
+  auto index = safeLabel.find(".nxs");
```

Comment:
This hard codes .nxs as the extension, don't you really mean any extension?

Reply:
it could be any extension yes, but it is always nxs (at present)